lib/HighlightsFeed.jsm

```diff
@@ -280,10 +252,10 @@ this.HighlightsFeed = class HighlightsFeed {
         this.fetchHighlights({broadcast: true});
         break;
       case at.DELETE_FROM_POCKET:
-        this.deleteFromPocket(action.data.pocket_id);
+        this.fetchHighlights({broadcast: true});
```

Comment:
I think it's confusing why a `DELETE_FROM_POCKET` case fires off a `fetchHIghlights`

Reply:
This doesn't impact DS but in Activity stream both delete and archive from pocket actions should trigger refresh of highlight content so that we don't show deleted and archived pocket links in highlights.

Reply:
Ah yes, I see these lines inside the deleted highlights code.

One thought though is maybe a comment that you're just forcing a refresh after something has changed.